salt/hg/config/hg.apache.conf.jinja

```diff
@@ -1,5 +1,13 @@
+#Configure rate limiting
+QS_ClientEventLimitCount 10 60 rate_limit
```

Comment:
This looks great! I think the one suggestion I would have is that the "default" global rate limit be more permissive, maybe 120 per 60 seconds. We can then configure `QS_ClientEventPerSecLimit` inside a given `VirtualHost`